digit-health-services/libraries/health-services-common/src/main/java/org/digit/health/common/data/query/builder/UpdateQueryBuilder.java

```diff
@@ -0,0 +1,19 @@
+package org.digit.health.common.data.query.builder;
+
+import java.util.List;
+
+public class UpdateQueryBuilder implements GenericQueryBuilder{
+    @Override
+    public String build(Object object) {
+        StringBuilder queryStringBuilder = null;
+        try {
+            String tableName = GenericQueryBuilder.getTableName(object.getClass());
+            List<String> fieldsToUpdate = GenericQueryBuilder.getFieldsWithCondition(object,  QueryFieldChecker.isNotNull);
+            List<String> fieldsToUpdateWith = GenericQueryBuilder.getFieldsWithCondition(object,  QueryFieldChecker.isAnnotatedWithUpdateBy);
+            queryStringBuilder = GenericQueryBuilder.generateQuery(GenericQueryBuilder.updateQueryTemplate(tableName), fieldsToUpdate, fieldsToUpdateWith);
+        } catch (Exception exception) {
+            System.out.println(exception.getMessage());
```

Comment:
Do the same thing here as well.
`throw new QueryBuilderException(exception.getMessage);`

Reply:
Created QueryBuilderException.